docs/ossm/istioctl/README.md

```diff
@@ -0,0 +1,69 @@
+[Return to OSSM Docs](../)
+
+# Installing the istioctl tool
+
+The `istioctl` tool is a configuration command line utility that allows service 
+operators to debug and diagnose Istio service mesh deployments.
+
+## Steps
+
+1. Download `istioctl` binary
+
+    In the OpenShift console, navigate to the Command Line Tools by clicking :grey_question: -> **Command Line Tools** in the upper-right of the header.  
+    Then click on **Download istioctl**, choose the right version and architecture according to your system.
+
+1. Extract the `istioctl` binary.
+
+    ```bash
+    tar xzf istioctl-<VERSION>-<OS>-<ARCH>.tar.gz
+    ```
+
+1. Move to the uncompressed directory.
+
+    ```bash
+    cd istioctl-<VERSION>-<OS>-<ARCH>
+    ```
+
+1. Add `istioctl` client to your path.
+
+    ```bash
+    export PATH=$PATH:$PWD
```

Comment:
Since we want the local copy of `istioctl` to be used, otherwise any existing `istioctl` in the $PATH, it will be used. 

```suggestion
    export PATH=$PWD:$PATH
```

Reply:
Good point ! Changed